getting-started/user-metadata.mdx

```diff
@@ -1,54 +1,85 @@
 ---
-title: "User Metadata"
+title: "User Properties"
 ---
 
-By default, the only required field a user needs is an email address. There are a few more options to customize the
-user experience. They can be found under **User Schema** in the sidebar.
+User properties allows you to define the fields that make up your user object.
+These fields could be attributes like a **name**, which we will collect on signup,
+or a **profile picture**, which we will allow users to upload, or even a custom internal field like a **Stripe customer id**.
 
-Enabling an extra field has a few effects:
+![User Properties](/images/user-properties.png)
 
-- Your signup page will now also ask for that field (except for profile picture,
-  where we start them off with a default).
+## Types of User Properties
 
-![enable name](/images/configure-4.png)
+There are two types of user properties:
 
-- Users that sign in via passwordless login (which only asks for email address),
-  will be asked to provide the extra field when they first sign up.
-- Existing users will be asked to fill in that field when they next sign up.
-- Your users can update those fields on your hosted account pages.
+ - **Managed by your users** - These are fields that can be viewed and edited by your users. Some examples of these are **name**, **profile picture**, **terms of service**, etc.
+   You and your team can also edit these fields on behalf of your users.
```

Comment:
Should specify how they can manage these fields on behalf of their users (via the dashboard? APIs?)